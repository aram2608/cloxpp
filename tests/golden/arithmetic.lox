print 1 + 2 * 3;       // expect 7
print (1 + 2) * 3;     // expect 9
print 10 / 2 - 3;      // expect 2
print -5 + 8;          // expect 3