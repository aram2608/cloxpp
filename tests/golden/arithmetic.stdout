Token: PRINT print 1
Token: NUMBER 1 1
Token: PLUS + 1
Token: NUMBER 2 1
Token: STAR * 1
Token: NUMBER 3 1
Token: SEMICOLON ; 1
Token: PRINT print 2
Token: LEFT_PAREN ( 2
Token: NUMBER 1 2
Token: PLUS + 2
Token: NUMBER 2 2
Token: RIGHT_PAREN ) 2
Token: STAR * 2
Token: NUMBER 3 2
Token: SEMICOLON ; 2
Token: PRINT print 3
Token: NUMBER 10 3
Token: SLASH / 3
Token: NUMBER 2 3
Token: MINUS - 3
Token: NUMBER 3 3
Token: SEMICOLON ; 3
Token: PRINT print 4
Token: MINUS - 4
Token: NUMBER 5 4
Token: PLUS + 4
Token: NUMBER 8 4
Token: SEMICOLON ; 4
Token: EOF  4
